packages/transport-websockets/test/node.ts

```diff
@@ -9,13 +9,15 @@ import { multiaddr } from '@multiformats/multiaddr'
 import { WebSockets, WebSocketsSecure } from '@multiformats/multiaddr-matcher'
 import { expect } from 'aegir/chai'
 import { isLoopbackAddr } from 'is-loopback-addr'
+import { createLibp2p } from 'libp2p'
```

Comment:
This module doesn't depend on `libp2p` but adding this test here would mean the dependency would need to be added.

This is intentional as it creates weird dependency loops that mean (among other things) a patch release of `@libp2p/websockets` would also cause a patch release of `libp2p` even if no changes were made there.

If you need to do integration-style testing, tests should be added to the [integration-tests](https://github.com/libp2p/js-libp2p/tree/main/packages/integration-tests) module, this won't cause any cascading releases.